modules/nextflow/src/main/groovy/nextflow/scm/AwsCodeCommitCredentialProvider.groovy

```diff
@@ -0,0 +1,320 @@
+package nextflow.scm
```

Comment:
This source has been copied from the Spring framework lib, isn't it. It's very important to include the original license boilerplate text and author citation.